pkg/plugins/generic/generic_plugin.go

```diff
@@ -277,7 +278,11 @@ func needDriverCheckVdpaType(state *sriovnetworkv1.SriovNetworkNodeState, driver
 // editKernelArg Tries to add the kernel args via ostree or grubby.
 func editKernelArg(helper helper.HostHelpersInterface, mode, karg string) error {
```

Comment:
Will this logic really work in the systemd mode? How it will behaves?
1. Reboot
2. systemd service exec
3. systemd service edited the grub config
4. Host booted
5. sriov-daemon pod starts
6. sriov-daemon detects that running kernel is missing required args
7.  host reboot again
8. ...
9. stable state

Is the sequence above describes the flow properly?

Reply:
in systemd mode kernel arg are configured by sriov-service so it doesn't do reboot loop

Reply:
@e0ne wont kernel args be applied only after reboot ?
hence two reboots will be required for this case ? 

(unless we set kargs in daemon even in systemd mode)